packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -26,9 +41,179 @@ import { useFeatureFlag } from '../FeatureFlags';
 import { usePrefix } from '../../internal/usePrefix';
 import { FormContext } from '../FluidForm';
 
-const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
+export interface FilterableMultiSelectProps<Item extends ItemBase>
+  extends SortingPropTypes<Item> {
+  /**
+   * Specify a label to be read by screen readers on the container node
+   */
+  'aria-label'?: string;
+
+  className?: string;
+
+  /**
+   * Specify the direction of the multiselect dropdown.
+   */
+  direction?: 'top' | 'bottom';
+
+  /**
+   * Disable the control
+   */
+  disabled?: boolean;
+
+  /**
+   * Additional props passed to Downshift
+   */
+  downshiftProps?: DownshiftProps<Item>;
+
+  /**
+   * Default sorter is assigned if not provided.
+   */
+  filterItems(
```

Comment:
This one is optional too, but it seems that the proper way to get `intellisense` working when the given prop has a `defaultProps` entry is to omit the optionality

Reply:
We're going to begin moving away from defaultProps to use default function params, #13424  

If it helps with this, feel free to refactor this to use default function params

Reply:
Nice to hear that, but I'm not really eager to balloon this already quite complex refactor…

How does is sound to you to delegate this one to the task you linked, as far as it is not needed for the typescript addition itself (e.g. so that it doesn't produce type errors)